api/sspanel/sspanel.go

```diff
@@ -153,6 +153,7 @@ func (c *APIClient) parseResponse(res *resty.Response, path string, err error) (
 // GetNodeInfo will pull NodeInfo Config from sspanel
 func (c *APIClient) GetNodeInfo() (nodeInfo *api.NodeInfo, err error) {
 	path := fmt.Sprintf("/mod_mu/nodes/%d/info", c.NodeID)
+	var res, err string
```

Comment:
delete
```go
var res, err string
if etag, ok := c.eTag[NodeInfo]; ok {
res, err := c.client.R().
	SetResult(&Response{}).
	SetHeader("If-None-Match", etag).
	ForceContentType("application/json").
	Get(path)
} else {
res, err := c.client.R().
	SetResult(&Response{}).
	ForceContentType("application/json").
	Get(path)
}
```
just use
```go
res, err := c.client.R().
	SetResult(&Response{}).
	SetHeader("If-None-Match", c.eTag[NodeInfo]).
	ForceContentType("application/json").
	Get(path)
```
`res.StatusCode() == 304` can be work to determine the result

Reply:
Sorry, I just saw your reply now。
if c.eTag["NodeInfo"] is nil,Will it send an empty HTTP header If-None-Match?

Reply:
It's doesn't master. webapi can handle this problem.